### DISCUSSION TITLE
Pytorch Issue -- Ubuntu 22.04.1 LTS, Release: 22.04

### DISCUSSION THREAD

**Discussion by jjherlan** [cnr-isti-vclab/TagLab#77] [General]
Hello and Happy New Year!

I am running: 
Ubuntu 22.04.1 LTS, Release: 22.04

I have successfully installed CUDA v. 11.7.0-1 or 11.5 (below)?

nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Thu_Nov_18_09:45:30_PST_2021
Cuda compilation tools, release 11.5, V11.5.119
Build cuda_11.5.r11.5/compiler.30672275_0

But I get the following error message during installation:

Found NVCC version: 11.5
nvcc version installed not supported by pytorch!!
Something is wrong with NVCC. Do you want to install the CPU version of pythorch? [Y/n]

Is there a solution to install the GPU version of pytorch yet?

Thank you, in advance.

James

**Comment by cappelletto** [comment]
Hi James, did you manage to solve this? 
I had a similar issue when upgrading to Ubuntu 22.04. Could you post your current environment setup?

`pip list`

Cheers,
J

**Comment by jjherlan** [comment]
Sorry to be MIA, just getting back to this project:

`pip list`

absl-py                      1.4.0
affine                       2.4.0
albumentations               1.3.1
astunparse                   1.6.3
attrs                        23.1.0
boltons                      23.0.0
brotlipy                     0.7.0
cachetools                   5.3.1
certifi                      2023.5.7
cffi                         1.15.1
charset-normalizer           2.0.4
click                        8.1.3
click-plugins                1.1.1
cligj                        0.7.2
conda                        23.5.0
conda-content-trust          0.1.3
conda-package-handling       2.0.2
conda_package_streaming      0.7.0
contourpy                    1.1.0
cryptography                 39.0.1
cycler                       0.11.0
filelock                     3.9.0
flatbuffers                  23.5.26
fonttools                    4.40.0
fsspec                       2023.4.0
gast                         0.4.0
GDAL                         3.4.1
google-auth                  2.21.0
google-auth-oauthlib         1.0.0
google-pasta                 0.2.0
grpcio                       1.56.0
h5py                         3.9.0
idna                         3.4
imageio                      2.31.1
jax                          0.4.13
Jinja2                       3.1.2
joblib                       1.3.1
jsonpatch                    1.32
jsonpointer                  2.1
keras                        2.12.0
kiwisolver                   1.4.4
lazy_loader                  0.3
libclang                     16.0.0
Markdown                     3.4.3
MarkupSafe                   2.1.2
matplotlib                   3.7.2
ml-dtypes                    0.2.0
mpmath                       1.2.1
networkx                     3.0rc1
numpy                        1.22.3
oauthlib                     3.2.2
opencv-python                4.8.0.74
opencv-python-headless       4.8.0.74
opt-einsum                   3.3.0
packaging                    23.0
pandas                       2.0.3
Pillow                       9.3.0
pip                          23.1.2
pluggy                       1.0.0
protobuf                     4.23.3
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pycocotools                  2.0.6
pycosat                      0.6.4
pycparser                    2.21
pyOpenSSL                    23.0.0
pyparsing                    3.0.9
PyQt5                        5.15.9
PyQt5-Qt5                    5.15.2
PyQt5-sip                    12.12.1
PySocks                      1.7.1
python-dateutil              2.8.2
pytorch-triton               2.1.0+440fd1bf20
pytz                         2023.3
PyWavelets                   1.4.1
PyYAML                       6.0
qudida                       0.0.4
rasterio                     1.3.8
requests                     2.28.1
requests-oauthlib            1.3.1
rsa                          4.9
ruamel.yaml                  0.17.21
ruamel.yaml.clib             0.2.6
scikit-image                 0.21.0
scikit-learn                 1.3.0
scipy                        1.11.1
setuptools                   68.0.0
shapely                      2.0.1
six                          1.16.0
sklearn                      0.0.post5
snuggs                       1.4.7
sympy                        1.11.1
tensorboard                  2.12.3
tensorboard-data-server      0.7.1
tensorflow                   2.12.0
tensorflow-estimator         2.12.0
tensorflow-io-gcs-filesystem 0.32.0
termcolor                    2.3.0
threadpoolctl                3.1.0
tifffile                     2023.7.4
toolz                        0.12.0
torch                        2.1.0.dev20230701+cu121
torchaudio                   2.1.0.dev20230702+cu121
torchvision                  0.16.0.dev20230702+cu121
tqdm                         4.65.0
typing_extensions            4.4.0
tzdata                       2023.3
urllib3                      1.26.15
Werkzeug                     2.3.6
wheel                        0.40.0
wrapt                        1.14.1
zstandard                    0.19.0

**Comment by alemuntoni** [comment]
Hello everyone, 
could you please try to type `nvcc --version` in a terminal and paste the output here?
It seems that the default installation of cuda is 11.5, which is not supported. Maybe the 11.7 installation is just hidden by 11.5.

**Reply by jjherlan** [reply]
Thank you for the reply. I now have Cuda 11.8. I do not think that is supported yet, could you confirm?

`nvcc --version`

nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:33:58_PDT_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0
